modules/util/config/TrainConfig.py

```diff
@@ -222,6 +222,10 @@ class TrainConfig(BaseConfig):
 
     # training settings
     learning_rate_scheduler: LearningRateScheduler
+    custom_scheduler: str | None
```

Comment:
There are two types of schedulers commonly used in diffusion training: learning rate schedulers and noise schedulers. I'm always trying to be explicit about the type, which means the name should either contain `learning_rate_scheduler` or `noise_scheduler`. Please change these names to reflect this naming convention

Reply:
Changed to `custom_learning_rate_scheduler`.